src/renderer/utils/markdown.ts

```diff
@@ -130,6 +130,84 @@ const Markdown = {
 
     },
 
+    // Wikilink
+    wikilink () {
+
+      const {path: notesPath, re} = Config.notes;
+
+      if ( !notesPath ) return [];
+
+      var matches = [];
+      var codeblocks = [];
+      return [
+        { // Extract Codeblocks
```

Comment:
Detecting code blocks early on is probably a pretty good approach, if it's possible to do so reasonably easily.

I'll try to get this merged in v1.3.0 👍